modules/margin/src/main/java/com/opengamma/sdk/margin/v3/InvokerMarginClient.java

```diff
@@ -216,6 +221,57 @@ public MarginCalcResult calculate(Ccp ccp, MarginCalcRequest request) {
   }
 
   @Override
+  public MultiCcpMarginCalcResult calculate(List<Ccp> ccps, MarginCalcRequest request) {
+
+    HashMap<Ccp, ScheduledFuture<MarginCalcResult>> futures = new HashMap<>();
+    for (Ccp ccp : ccps) {
+      Callable<MarginCalcResult> ccpAsyncCalculator = () -> calculate(ccp, request);
+      ScheduledFuture<MarginCalcResult> future = invoker.getExecutor().schedule(ccpAsyncCalculator, 5, TimeUnit.SECONDS);
```

Comment:
This code sleeps for 5 seconds before calculating, you just want `submit()` I think